issue found on: server/server.go:54:2: importShadow

an issue has been detected in source code: ./server/server.go:54:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-results-aggregator-mock/groups' package 'groups'

--- a/server/server.go
+++ b/server/server.go
@@ -51,12 +51,12 @@ type HTTPServer struct {
 // New constructs new implementation of Server interface
 func New(config Configuration,
 	storageInstance storage.Storage,
-	groups map[string]groups.Group,
+	ruleGroups map[string]groups.Group,
 	ruleContents []content.RuleContent) *HTTPServer {
 	return &HTTPServer{
 		Config:  config,
 		Storage: storageInstance,
-		Groups:  groups,
+		Groups:  ruleGroups,
 		Content: ruleContents,
 	}
 }

